ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -406,6 +407,7 @@ public void setAllowFontScaling(ReactEditText view, boolean allowFontScaling) {
   @ReactProp(name = "placeholder")
   public void setPlaceholder(ReactEditText view, @Nullable String placeholder) {
     view.setHint(placeholder);
+    view.setEllipsize(TextUtils.TruncateAt.END);
```

Comment:
Awesome. Does the multiline placeholder work on multiline textinput? :peace_symbol: :pray: 

lets say I have very long text input with very long description...